src/service/rpc/generic/LoadBalancer.ts

```diff
@@ -543,69 +543,75 @@ export class LoadBalancer implements AbstractRpcInterface {
     }
   }
 
-  async post<T>({ path, body, prefix }: PostI): Promise<T> {
+  modifyNodeUrl(url: string) {
+    return url
```

Comment:
This function do nothing 🤔

Reply:
It is here for ability to tweak the url. See bottom of the file. 
Yes in 99% of usages just use same value, but for the TronLoadBalancer it is changed.

Reply:
Ahh, got it.